# AE 01: Meet the penguins
## Greg Chism

For this application exercise, we'll use the pandas and palmerpenguins packages.

In [2]:
import pandas as pd
from palmerpenguins import load_penguins

First, let's read in the data.

In [4]:
penguins = load_penguins()

The dataset we will visualize is called `penguins`.
Let's `.info()` at it.

In [5]:
2 * 4

8

-   **Your turn:** Replace `#add code here` with the code for "info"ing at the data `penguins` data frame – `penguins.info()`. Render the document and view the output.

In [6]:
penguins.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 344 entries, 0 to 343
Data columns (total 8 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   species            344 non-null    object 
 1   island             344 non-null    object 
 2   bill_length_mm     342 non-null    float64
 3   bill_depth_mm      342 non-null    float64
 4   flipper_length_mm  342 non-null    float64
 5   body_mass_g        342 non-null    float64
 6   sex                333 non-null    object 
 7   year               344 non-null    int64  
dtypes: float64(4), int64(1), object(3)
memory usage: 21.6+ KB


-   **Demo:** First, replace the blank below with the number of rows in the `penguins` data frame based on the output of the chunk below. Then, replace it with "inline code" and render again.

In [7]:
len(penguins)

344

There are 344 penguins in the `penguins` data frame.